src/Nethermind/Nethermind.Specs/ChainSpecStyle/AuRaParameters.cs

```diff
@@ -67,6 +68,10 @@ public class AuRaParameters
 
         public IDictionary<long, IDictionary<Address, byte[]>> RewriteBytecode { get; set; }
 
+        public Address WithdrawalContractAddress { get; set; }
+
+        public ForkActivation WithdrawalContractTransition { get; set; }
```

Comment:
do we really need this `WithdrawalContractTransition`? I would say withdrawals on gnosis come with a withdrawal contract which is activated at the time we activate the withdrawals fork. Basically instead of looking at `WithdrawalContractTransition` let's just look at whatever we just for withdrawals activation on the client (for mainnet), no need to introduce it as AuRa specific imho.

Reply:
True, should be handled by the EipTransition.